<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>461</height>
     </rect>
    </property>
    <widget class="QWidget" name="auth">
     <widget class="QPushButton" name="authButton">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Auth</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="passText2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>190</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="idText2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>80</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="reg">
     <widget class="QPushButton" name="registration">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Reg</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="idText">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="passText">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="message">
     <widget class="QLineEdit" name="to_user">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="message_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>280</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="send">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
